### PR TITLE
Feature/pmcp 4438 : Modify Mega menu footer and item twigs. 

### PR DESCRIPTION
**Pull request by rutviksavsani**
### Make sure you complete these items:

- [x] Updated root CHANGELOG.md with summary of changes under `Unpublished` section
- [x] `npm run prod` in this repo outputs expected changes (excepting the issue with re-ordered partials in larva-css algorithms partials - see [LRVA-1885](https://jira.pmcdev.io/browse/LRVA-1885))
- [ ] If adding a new pattern, in the PR comment, included a screenshot and link to the static Vercel deployment
- [ ] If changes to build scripts or the Node.js server, tested changes in pmc-spark [via a pre-release](https://confluence.pmcdev.io/x/XhOeAw)
- - If changes to build tools: npm scripts `prod`, `lint`, and `dev` scripts run as expected
- - If changes to Larva server: Static site generates as expected in a theme  (avail. on a URL {brand}.stg.larva.pmcdev.io)

### CODE REVIEW

File: packages/larva-patterns/modules/mega-menu-footer/mega-menu-footer.twig [hunk 1]
```diff
@@ -1,8 +1,11 @@
 <footer class="mega-menu__footer // {{ mega_menu_footer_classes }} ">
 	<div class="mega-menu__footer-inner // {{ mega_menu_footer_inner_classes }} ">
-		<section class="mega-menu__footer-subscribe // {{ mega_menu_footer_subscribe_classes }} ">
-			{% include "@larva/modules/mega-menu-footer-subscribe/mega-menu-footer-subscribe.twig" with mega_menu_subscribe %}
-		</section>
+
+		{% if mega_menu_subscribe %}
```
Comment (rsalvucci) [inline packages/larva-patterns/modules/mega-menu-footer/mega-menu-footer.twig:4]: This if check may cause regressions on other brands

File: packages/larva-patterns/modules/mega-menu-item/mega-menu-item.twig [hunk 2]
```diff
@@ -1,9 +1,15 @@
-<li class="mega-menu-item // mega-menu__parent-list-item {{ mega_menu_item_classes }} ">
+<li class="mega-menu-item // mega-menu__parent-list-item {{ mega_menu_item_classes }} " data-collapsible="collapsed" >
 	<div class="mega-menu__item-link // {{ mega_menu_item_link_classes }} ">
```
Comment (rsalvucci) [inline packages/larva-patterns/modules/mega-menu-item/mega-menu-item.twig:2]: Will this cause any regressions on other brands?

**Comment by rsalvucci** [comment]
Please remove the package-lock.json files.

File: packages/larva-patterns/modules/mega-menu-footer/mega-menu-footer.twig (hunk 1 above)
Comment (rutviksavsani) [reply packages/larva-patterns/modules/mega-menu-footer/mega-menu-footer.twig:4]: Checked with RS and looks good to me, Had to introduce this check as not all brands have subscribe card in mega menu.
SPY does not have the card hence, Introduced the condition.

File: packages/larva-patterns/modules/mega-menu-item/mega-menu-item.twig (hunk 2 above)
Comment (rutviksavsani) [reply packages/larva-patterns/modules/mega-menu-item/mega-menu-item.twig:2]: No, Verified on RS and this does not create any regression issues.
Future ref: https://penskemediacorp.slack.com/archives/C03RZKELNR5/p1664194743857379?thread_ts=1664187688.768309&cid=C03RZKELNR5

**Comment by rutviksavsani** [comment]
> Please remove the package-lock.json files.

**Review by rsalvucci — APPROVED** [review]

File: packages/larva-patterns/modules/mega-menu-item/mega-menu-item.twig [hunk 3]
```diff
@@ -1,9 +1,15 @@
...
+
+		{% if c_icon %}
+			<button class="mega_menu__item-button // {{ mega_menu_item_button_classes }}" data-collapsible-toggle="always-show">
+				{% include "@larva/components/c-icon/c-icon.twig" with c_icon %}
+			</button>
+		{% endif %}
 	</div>
 
-	<ul class="mega-menu__child-list // {{ mega_menu_child_list_classes }} ">
+	<ul class="mega-menu__child-list // {{ mega_menu_child_list_classes }} " data-collapsible-panel="" data-collapsible-breakpoint="mobile-only">
```
[the remark this answers is not in the document]
Comment (rutviksavsani) [reply packages/larva-patterns/modules/mega-menu-item/mega-menu-item.twig:12]: Checked that as well, no regression observed.

**Pull request merged**